fix(radio-button): adding checkmark inside radio for redesign

kdang-coveo opened the pull request:
### Proposed Changes
Fixing radio button to look like mockup design:

**Mockup**
![image](https://user-images.githubusercontent.com/66333175/106036717-adfe6c00-60a3-11eb-80ac-12801b5ed1e4.png)

![image](https://user-images.githubusercontent.com/66333175/106036736-b2c32000-60a3-11eb-83b0-2106a71738d0.png)

<!-- Explain what are your changes. -->
- Added checkmark svg in Radio button and adjusted alignment

**Implementation Results:**

![image](https://user-images.githubusercontent.com/66333175/106038459-f9197e80-60a5-11eb-8147-5f9cd9265da2.png)
![image](https://user-images.githubusercontent.com/66333175/106038484-ff0f5f80-60a5-11eb-92a6-eea7830a21fd.png)




### Potential Breaking Changes

<!-- List all changes that might be breaking to react-vapor's users if any. -->

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/react-vapor/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

aiheon commented on packages/vapor/scss/controls/radios.scss:
```diff
@@ -56,8 +57,10 @@
             &:after {
                 z-index: 0;
+                padding: 5px;
                 background-color: #1372ec; // digital-blue-60
```
this here can be changed to use the variable :+1:

aiheon commented on packages/vapor/scss/controls/radios.scss:
```diff
@@ -67,13 +70,19 @@
-                    background-color: var(--medium-grey);
+                    padding: 3px;
+                    background-color: var(--grey-20);
+                    border: 2px solid #e5e8e8;
```
here too I think :thinking:

guscoveo reviewed (changes requested):
Hi Kien, thanks for improving the alignment of the radio button! I fixed it in the library as well. There are some minor tweaks to add though. Please do slack me to let me know if ever it isn't clear 🙏 

![image](https://user-images.githubusercontent.com/77753668/106171709-e06d9f00-615f-11eb-8146-07ca12f71400.png)

rosalie-liu commented on packages/vapor/scss/controls/radios.scss:
```diff
@@ -12,8 +12,10 @@
+            padding-top: $radio-button-text-padding-top;
             padding-left: $radio-button-size + $checkbox-label-margin;
             color: var(--dark-grey);
+            font-weight: 400;
```
```suggestion
            font-weight: var(--main-font-bold);
```

rosalie-liu commented on packages/vapor/scss/controls/radios.scss:
```diff
@@ -67,13 +73,17 @@
-                    background-color: var(--medium-grey);
+                    padding: 3px;
+                    background-color: var(--grey-20);
+                    border: 2px solid #e5e8e8;
```
```suggestion
                    border: 2px solid var(--grey-40);
```

kdang-coveo commented:
> Hi Kien, thanks for improving the alignment of the radio button! I fixed it in the library as well. There are some minor tweaks to add though. Please do slack me to let me know if ever it isn't clear 🙏
> 
> ![image](https://user-images.githubusercontent.com/77753668/106171709-e06d9f00-615f-11eb-8146-07ca12f71400.png)

- [x] Corrected Radio button Border colour
- [x] Font weight 400 & 500 applied with var
- [x] Colour checkmark fixed to correct colour
- [ ] Transition effect - slightly left due to padding - will explore a different svg or reposition

aiheon commented:
@kdang-coveo  the transition effect you talk about, is it the label moving up and down because of the font weight changing? :thinking:  maybe the padding is not big enough in the demo xD 
![image](https://user-images.githubusercontent.com/63734941/106287117-719e4d80-6214-11eb-96cb-41520c9388e8.png)

kdang-coveo commented:
> @kdang-coveo the transition effect you talk about, is it the label moving up and down because of the font weight changing? 🤔 maybe the padding is not big enough in the demo xD
> ![image](https://user-images.githubusercontent.com/63734941/106287117-719e4d80-6214-11eb-96cb-41520c9388e8.png)

The transition effect mentioned by @guscoveo, was when it was expanding while checked, it expanded slightly to the left and out. I believe it was a padding issue but P.O resolved this issue with his commit 👍 

![radio-transition](https://user-images.githubusercontent.com/66333175/106304074-e5e2ec00-6228-11eb-98f4-980fee12f26f.gif)

kdang-coveo commented:
New transition effect:
![radio transition fixed](https://user-images.githubusercontent.com/66333175/106655261-20b98c80-6567-11eb-997a-726abffdc2b2.gif)
